libr/main/radare2.c

```diff
@@ -1328,8 +1328,14 @@ R_API int r_main_radare2(int argc, char **argv) {
 		r_flag_space_set (r.flags, NULL);
 		/* load <file>.r2 */
 		{
-			char f[128];
-			snprintf (f, sizeof (f), "%s.r2", pfile);
+			char* f = r_str_newf("%s.r2", pfile);
```

Comment:
f is never freed, so its leaking, also, missing sspace before (

Reply:
It seems the syntax checker script is missing those.